### ISSUE TITLE
Generated partial classes should match access modifiers

### ISSUE THREAD

**Issue by paulloz** [31/GodotOnReady#50]
The following code will not compile using GodotOnReady. The generated partial class is going to be `public`, raising [CS0262](https://learn.microsoft.com/en-us/dotnet/csharp/misc/cs0262).
```cs
internal partial class Foo : Node
{
    [OnReadyGet("Foo")]
    private Node foo;
}
```

**Comment by 31** [comment]
Good catch. Best way to fix is to actually not have any access modifiers, IIRC.

**Comment by 31** [comment]
Fixed in 1.2.3. Validation in progress on the NuGet gallery.

**Comment by 31** [comment]
Done, and thanks, by the way. 😄
